### ISSUE TITLE
Translatable timezones

### ISSUE THREAD

**Issue by pzmarzly** [techouse/intl-date-time#21]
It would be nice to be able to change the way timezones are displayed, by providing translations.

```diff
-<span v-if="field.displayUserTimeZone" class="text-80 text-sm ml-2">({{ userTimezone }})</span>
+<span v-if="field.displayUserTimeZone" class="text-80 text-sm ml-2">({{ __(userTimezone) }})</span>
```

Then we would be able to, for example, present the user with `(UK time)` instead of `(Europe/London)`, by putting:

```js
"Europe/London": "UK time"
```

into `resources/lang/vendor/nova/en.json`.

I'm not sure whether `__` is available as Vue helper out of the box, if not, it can be easily implemented like so:

```js
let __ = function (original) {
    let translated = window.config.translations[original];
    if (translated !== undefined) return translated;
    return original;
}
```

**Comment by techouse** [comment]
Thanx for the feedback. `__` is not available out of the box, however, one can use something like [Vue I18n](https://kazupon.github.io/vue-i18n/) which uses the function call `$t('string to translate')`.

Feel free to make a PR if you'd like. :) Please be sure to test it before submitting it.

**Comment by techouse** [comment]
Implemented in `v1.5.0`
